src/main/java/io/confluent/idesidecar/restapi/models/ConnectionSpec.java

```diff
@@ -236,9 +235,16 @@ public record CCloudConfig(
           description = "The identifier of the CCloud organization to use. "
                         + "The user's default organization is used when absent."
       )
-      @JsonProperty(value = "organization_id", required = true)
```

Comment:
Since the CCloudConfig now holds multiple properties, I think it doesn't make sense to keep requiring the CCloud org to be set, as we'll use the user's default org in the absence of this property per the docs.